infrastructure/ansible/dynamic.inventory/TO.py

```diff
@@ -0,0 +1,241 @@
+            if flat_server_profile not in out.keys():
+                out['server_profile']['children'].append(flat_server_profile)
+                out[flat_server_profile] = self.populate_server_profile_vars(
+                    traffic_ops_api,
+                    server['profileId'])
+            out[flat_server_profile]['hosts'].append(fqdn)
+            if flat_cachegroup not in out.keys():
+                out['cachegroup']['children'].append(flat_cachegroup)
+                # out[flat_cachegroup] = self.populate_cachegroups(
+                cgdata = self.populate_cachegroups(
+                    traffic_ops_api,
+                    server['cachegroupId'])
+                out[flat_cachegroup] = cgdata.cgvars
+                flat_parent_cg = cgdata.primary_parent_group_name
+                flat_second_parent_cg = cgdata.secondary_parent_group_name
+                if flat_parent_cg not in out.keys():
+                    out[flat_parent_cg] = {}
+                    out[flat_parent_cg]['children'] = []
+                if flat_second_parent_cg not in out.keys():
+                    out[flat_second_parent_cg] = {}
+                    out[flat_second_parent_cg]['children'] = []
+                out[flat_parent_cg]['children'].append(flat_cachegroup)
+                out[flat_second_parent_cg]['children'].append(flat_cachegroup)
+            out[flat_cachegroup]['hosts'].append(fqdn)
+            if flat_server_type not in out.keys():
+                out['server_type']['children'].append(flat_server_type)
+                out[flat_server_type] = {}
+                out[flat_server_type]['hosts'] = []
+            out[flat_server_type]['hosts'].append(fqdn)
+            if flat_server_cdn_name not in out.keys():
+                out['server_cdnName']['children'].append(flat_server_cdn_name)
+                out[flat_server_cdn_name] = {}
+                out[flat_server_cdn_name]['hosts'] = []
+            out[flat_server_cdn_name]['hosts'].append(fqdn)
+            if flat_server_status not in out.keys():
+                out['server_status']['children'].append(flat_server_status)
+                out[flat_server_status] = {}
+                out[flat_server_status]['hosts'] = []
+            out[flat_server_status]['hosts'].append(fqdn)
+        return out
+
+    def to_inventory(self):
+        return self.generate_inventory_list(self.to_url)
+
+
+if __name__ == "__main__":
+    PARSER = argparse.ArgumentParser(
+        description='Generate ansible inventory from TrafficOps')
+
+    PARSER.add_argument(
+        '-to',
+        nargs='+',
+        help='Please pass username and password for TrafficOps')
+    PARSER.add_argument(
+        '-url',
+        type=str,
+        help='TO URL')
+    PARSER.add_argument('--list', action='store_true')
+    PARSER.add_argument('--host', type=str, help='Target TO Server')
+
+    ARGS = PARSER.parse_args()
+
+    TO_USER = os.getenv("TO_USERNAME")
```

Comment:
The variable used by `compare`, `genConfigRoutes.py`, `toaccess` (and the Python client's `toaccess` implementation), `ORT.py`, CiaB, and maybe others is `TO_USER` - it'd be nice to standardize so all the tools can use the same configuration if desired.

Reply:
Addressed with commit.